pkg/reconciler/pipelinerun/pipelinerun_test.go

```diff
@@ -6453,33 +6513,93 @@ func (prt PipelineRunTest) reconcileRun(namespace, pipelineRunName string, wantE
 	return reconciledRun, clients
 }
 
+func TestReconcile_RemotePipelineRefValidation_Failure(t *testing.T) {
+	names.TestingSeed()
+
+	namespace := "foo"
+	prName := "test-pipeline-run-failure"
+
+	prs := []*v1beta1.PipelineRun{parse.MustParseV1beta1PipelineRun(t, fmt.Sprintf(`
+metadata:
+  name: %s
+  namespace: %s
+spec:
+  pipelineRef:
+    resolver: bar
+  serviceAccountName: test-sa
+  timeout: 1h0m0s
+`, prName, namespace))}
+	ps := parse.MustParseV1beta1Pipeline(t, fmt.Sprintf(`
+metadata:
+  name: invalid-pipeline-no-params-defined
+  namespace: %s
+spec:
+  tasks:
+  - name: unit-test-1
+    taskSpec:
+      steps:
+        - name: my-steps
+          image: my-image
+          script: "$(params.doesnotexist)"
+`, namespace))
+
+	pipelineBytes, err := yaml.Marshal(ps)
```

Comment:
why not just `pipelineBytes := []byte(pipelineYaml)`? Is `ps` used anywhere else?

Reply:
fixed